src/main/resources/application.properties

```diff
@@ -1,4 +1,16 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/ajax
 spring.datasource.username=root
 spring.jpa.generate-ddl=true
-spring.jpa.hibernate.ddl-auto = create
+spring.jpa.hibernate.ddl-auto=update
+
+
+
+spring.data.mongodb.host=localhost
+spring.data.mongodb.port=27017
+spring.data.mongodb.database=ajax
+spring.main.allow-bean-definition-overriding=true
+
+
+
+
+
```

Comment:
to many line breaks :)

Reply:
why the discussion was resolved without changes in code?

Please, resolve discussions when you applied changes